software.md

```diff
@@ -0,0 +1,31 @@
+# FOSS United Community Software Policy
+This policy is devised to guide, but not mandate, software choices within the 
+FOSS United Community.
```

Comment:
The scope of this policy is not clear. Is this meant to be used when a new software is evaluated and adopted for use by the community like we did with BigBlueButton?

Reply:
Yes that's the intended purpose.